src/parser/contexts.h

```diff
@@ -1777,11 +1778,34 @@ struct ParseDefsCtx : TypeParserCtx<ParseDefsCtx> {
     }
     contents = contents.substr(lineSize + 1);
 
-    lexer = Lexer(contents);
+    auto colSize = contents.find(':');
+    auto rest = std::string_view();
```

Comment:
Is `rest` needed here? The code above seems to keep updating `contents` to look further up - can the same pattern not be used here as well?